src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -339,7 +339,12 @@ std::vector<std::string> disabledTestPatterns() {
     retVector.emplace_back(R"(MultipleLSTMCellTest/MultipleLSTMCellTest.CompareWithRefs.*)");
     // int8 / code-generation specific
     retVector.emplace_back(R"(smoke_LPT.*)");
-    retVector.emplace_back(R"(smoke_Snippets.*)");
+    retVector.emplace_back(R"(smoke_Snippets(?!_Eltwise).*)");
+    // arm snippets doesn't support sve_128 that required by dnnl injector jit_uni_eltwise_injector_f32 yet
```

Comment:
Could you please elaborate? Seems like subgraph bode in this test contains nodes that aren't supported by snippets on arm for now (emitters are not implemented)?

Reply:
`retVector.emplace_back(R"(smoke_Snippets(?!_Eltwise).*)");` retains smoke_Snippets_Eltwise tests, while still skipping other smoke_Snippets_xxx tests. smoke_Snippets_Eltwise contains the following test classes.
`Add`
`AddPair`
`AddConst`
`AddRollConst`
`EltwiseTwoResults` (skip)
`MaxNumParamsEltwise` (skip)
`ThreeInputsEltwise`
`TwoInputsAndOutputs` (skip)
`TwoInputsAndOutputsWithReversedOutputs` (skip)
These marked with (skip) have been skipped separately here https://github.com/openvinotoolkit/openvino/pull/23180#discussion_r1524301322. The others only contain Eltwise ops (Add Multiply Relu) that have been supported on ARM. Please feel free to have further discussions. Thanks Alexandra!

Reply:
Thanks Ivan for the comment! Before this PR, all smoke_Snippets test cases will be skipped for arm. As now this PR only support some Eltwise ops, only smoke_Snippets_Eltwise will be needed, we still need to skip all the other smoke_Snippets test cases on arm. So I revised smoke_Snippets  to smoke_Snippets(?!_Eltwise). I've added brief description, as currently we only have ticket for Snippets of Eltwise ops. Further discussion are always welcomed!